includes/tracks/class-wc-tracks.php

```diff
@@ -11,9 +11,14 @@
 class WC_Tracks {
 
 	/**
-	 * Tracks event name prefix.
+	 * Event name prefix for admin tracks.
 	 */
-	const PREFIX = 'wcadmin_';
+	const ADMIN_PREFIX = 'wcadmin_';
+
+	/**
+	 * Event name prefix for front-end tracks.
+	 */
+	const FRONTEND_PREFIX = 'wcstore_';
```

Comment:
Is this different from `woocommerceanalytics`?

Reply:
Happy to change it to `woocommerceanalytics`. That seemed long and unclear to me, and not consistent with `wcadmin_` :)